### ISSUE TITLE
SlevomatCodingStandard.Variables.UnusedVariable -- false positive

### ISSUE THREAD

**Issue by ghnp5** [slevomat/coding-standard#865]
Just a quick example:

```php
function test(int $test) : void {
	$result = mysqli_iquery('SELECT * FROM ...');
	if ($row = mysqli_fetch_assoc($result)) {
		$test = $row['cid'];
		
		do {
			test2($row);
		} while ($row = mysqli_fetch_assoc($result));
	}
}
```

The line with the "} while ($row", this "$row" variable is marked as UnusedVariable.

> [SlevomatCodingStandard.Variables.UnusedVariable.UnusedVariable]
> Unused variable $row.

**Comment by ghnp5** [comment]
Hey! Any luck on this one? :)
Thanks!

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/f7bd80b208302b7576b69c5ea0b5dfada362741b
